compute/cloud-client/src/main/java/compute/disks/consistencygroup/RemoveDiskFromConsistencyGroup.java

```diff
@@ -0,0 +1,93 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package compute.disks.consistencygroup;
+
+// [START compute_consistency_group_remove_disk]
+import com.google.cloud.compute.v1.Disk;
+import com.google.cloud.compute.v1.Operation;
+import com.google.cloud.compute.v1.RegionDisksClient;
+import com.google.cloud.compute.v1.RegionDisksRemoveResourcePoliciesRequest;
+import com.google.cloud.compute.v1.RemoveResourcePoliciesRegionDiskRequest;
+// If your disk has zonal location uncomment these lines
+//import com.google.cloud.compute.v1.DisksClient;
+//import com.google.cloud.compute.v1.DisksRemoveResourcePoliciesRequest;
+//import com.google.cloud.compute.v1.RemoveResourcePoliciesDiskRequest;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.concurrent.ExecutionException;
+
+public class RemoveDiskFromConsistencyGroup {
+
+  public static void main(String[] args)
+      throws IOException, ExecutionException, InterruptedException {
+    // TODO(developer): Replace these variables before running the sample.
+    // The project that contains the disk.
+    String project = "YOUR_PROJECT_ID";
+    // The zone or region of the disk.
+    String location = "us-central1";
+    // The name of the disk.
+    String diskName = "DISK_NAME";
+    // The name of the consistency group.
+    String consistencyGroupName = "CONSISTENCY_GROUP";
+    // The region of the consistency group.
+    String consistencyGroupLocation = "us-central1";
+    removeDiskFromConsistencyGroup(
+        project, location, diskName, consistencyGroupName, consistencyGroupLocation);
+  }
+
+  // Removes a disk from a Consistency Group.
+  public static Disk removeDiskFromConsistencyGroup(
+      String project, String location, String diskName,
+      String consistencyGroupName, String consistencyGroupLocation)
+      throws IOException, ExecutionException, InterruptedException {
+    String consistencyGroupUrl = String.format(
+        "https://www.googleapis.com/compute/v1/projects/%s/regions/%s/resourcePolicies/%s",
+        project, consistencyGroupLocation, consistencyGroupName);
+    // If your disk has zonal location uncomment these lines
+    //      try (DisksClient disksClient = DisksClient.create()) {
+    //        RemoveResourcePoliciesDiskRequest request =
+    //            RemoveResourcePoliciesDiskRequest.newBuilder()
+    //                .setDisk(diskName)
+    //                .setDisksRemoveResourcePoliciesRequestResource(
+    //                    DisksRemoveResourcePoliciesRequest.newBuilder()
+    //                        .addAllResourcePolicies(Arrays.asList(consistencyGroupUrl))
+    //                        .build())
+    //                .setProject(project)
+    //                .setZone(location)
+    //                .build();
```

Comment:
nit: consider creating a separate sample for zonal disk

Reply:
We are asked to implement all the logic in one sample. Created a check for zonal/regional location